### PR TITLE
feat(cli): add `warp --symbol` flag

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

- This allows users to interact read warp routes from the registry with symbol identifier rather than chain/address

### Drive-by

Rename `wei` to `amount`

### Backward compatibility

No, outputs chain map

### Testing

- Manual

(single)
```sh
$ yarn hyperlane warp read --symbol EZETH
$ yarn hyperlane warp send --symbol EZETH
```

(multiple)
```sh
$ yarn hyperlane warp read --symbol USDC
$ yarn hyperlane warp send --symbol USDC
```

### CODE REVIEW

File: typescript/cli/src/commands/warp.ts [hunk 1]
```diff
@@ -100,42 +106,86 @@ export const configure: CommandModuleWithContext<{
...
+        warpCoreConfig.tokens.map((t) => [t.chainName, t.addressOrDenom!]),
+      );
+    } else if (chain && address) {
+      addresses = {
+        [chain]: address,
+      };
+    } else {
+      logGreen(`Please specify either a symbol or chain and address`);
+      process.exit(0);
+    }
```
Comment (nbayindirli) [inline typescript/cli/src/commands/warp.ts:172]: this seems to overlap with
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/b0828b3d0eb95921a8753bb0c08b36b37ca0af15/typescript/cli/src/send/transfer.ts#L119-L130

can we consolidate somehow?

**Review by nbayindirli — CHANGES_REQUESTED** [review]
this overlaps significantly with https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3943/files (still draft)

only feedback is
1. we should probably support 2+ assets with same name (test for ETH on `yourchain` and `fuji` returned `No (or multiple) warp routes found for symbol ETH`),
2. should consider moving symbol logic into shared fn for use throughout cli

**Comment by nbayindirli** [comment]
> this overlaps significantly with https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3943/files (still draft)
> 
> only feedback is
> 
> 1. we should probably support 2+ assets with same name (test for ETH on `yourchain` and `fuji` returned `No (or multiple) warp routes found for symbol ETH`),
> 2. should consider moving symbol logic into shared fn for use throughout cli

I see 1 is addressed now so 2 is only concern

File: typescript/cli/src/commands/warp.ts (hunk 1 above)
Comment (nbayindirli) [reply typescript/cli/src/commands/warp.ts:172]: cc @jmrossy since we discussed this in-depth last week

**Review by ltyu — APPROVED** [review]

File: typescript/cli/src/commands/warp.ts (hunk 1 above)
Comment (yorhodes) [reply typescript/cli/src/commands/warp.ts:172]: I will add the same symbol option to this command though

**Review by nbayindirli — APPROVED** [review]
🚢

**Comment by nbayindirli** [comment]
may be an issue parsing `cli/test-configs/anvil/deployments/warp_routes/FAKE/anvil1-anvil2-config.yaml` when running `warp send` in ci-test now

**Pull request merged**